action-scripts/brownie/scripts/script_utils.py

```diff
@@ -91,12 +94,12 @@ def convert_output_into_table(outputs: list[dict]) -> str:
     Converts list of dicts into a pretty table
     """
     # Headers without "chain"
-    header = [k for k in outputs[0].keys()]
+    header = [k for k in outputs[0].keys() if k != "chain"]
```

Comment:
You don't want to have chain name in the table anymore?

Reply:
Nah.  It's redundant as it's also up top and in the filename.  I'm convinced.  And the tables are too wide.

Reply:
Ok, makes sense